src/ORM/Table.php

```diff
@@ -1787,7 +1787,13 @@ protected function _dynamicFinder($method, $args)
                 ));
             }
             foreach ($fields as $field) {
-                $conditions[$this->aliasField($field)] = array_shift($args);
+                $value = array_shift($args);
+
+                if (is_array($value)) {
+                    $field .= ' IN';
```

Comment:
This is dangerous as if an end user controls the values they can manipulate queries into finding multiple rows when the application is expecting a single row.